src/components/ShoppingCart/ShoppingCart.jsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react"
+import { Link } from "react-router-dom"
+import "./ShoppingCart.css"
+
+export default function ShoppingCart() {
+  return (
+    <nav className="shoppingcart">
```

Comment:
Will there ever be multiple shopping carts? Should this be an Id instead?

Reply:
There would only be one shopping cart, but it would contain multiple items inside this element. Similar to the pantry page vs the product card